metadata-ingestion/src/datahub/ingestion/source/sac/sac.py

```diff
@@ -0,0 +1,808 @@
+                resource_subtype=entity.resourceSubtype,
+                story_id=entity.storyId,
+                name=name,
+                description=entity.description.strip(),
+                created_time=entity.createdTime,
+                created_by=created_by,
+                modified_time=entity.modifiedTime,
+                modified_by=modified_by,
+                open_url=entity.openURL,
+                ancestor_path=ancestor_path,
+                is_mobile=entity.isMobile,
+                resource_models=resource_models,
+            )
+
+    def get_import_data_model_ids(self) -> Set[str]:
+        response = self.session.get(
+            url=f"{self.config.tenant_url}/api/v1/dataimport/models"
+        )
+        response.raise_for_status()
+
+        import_data_model_ids = set(
+            model["modelID"] for model in response.json()["models"]
+        )
+        return import_data_model_ids
+
+    def get_import_data_model_columns(
+        self, model_id: str
+    ) -> List[ImportDataModelColumn]:
+        response = self.session.get(
+            url=f"{self.config.tenant_url}/api/v1/dataimport/models/{model_id}/metadata"
+        )
+        response.raise_for_status()
+
+        model_metadata = response.json()
+
+        columns: List[ImportDataModelColumn] = []
+        for column in model_metadata["factData"]["columns"]:
+            columns.append(
+                ImportDataModelColumn(
+                    name=column["columnName"].strip(),
+                    description=column["descriptionName"].strip(),
+                    property_type=column["propertyType"],
+                    data_type=column["columnDataType"],
+                    max_length=column.get("maxLength"),
+                    precision=column.get("precision"),
+                    scale=column.get("scale"),
+                    is_key=column["isKey"],
+                )
+            )
+
+        return columns
+
+    def get_query_name(self, query: str) -> str:
+        if not self.config.query_name_template:
+            return query
+
+        query_name = self.config.query_name_template
+        query_name = query_name.replace("{name}", query)
+
+        return query_name
+
+    def get_view_name(self, schema: str, namespace: Optional[str], view: str) -> str:
+        if namespace:
+            return f"{schema}.{namespace}::{view}"
+
+        return f"{schema}.{view}"
+
+    def get_schema_field_data_type(
+        self, property_type: str, data_type: str
+    ) -> SchemaFieldDataTypeClass:
+        if property_type == "DATE":
+            return SchemaFieldDataTypeClass(type=DateTypeClass())
+        else:
+            if data_type == "string":
+                return SchemaFieldDataTypeClass(type=StringTypeClass())
+            elif data_type in ("decimal", "int32"):
+                return SchemaFieldDataTypeClass(type=NumberTypeClass())
+            else:
+                logger.warning(f"Unknown data type {data_type} found")
```

Comment:
```suggestion
                logger.info(f"Unknown data type {data_type} found")
```

Reply:
I have replaced the log with report_warning -> I think it is better to have this as a warning...usually there are not many possibilities for the data types in import data models - in case there will be ever an unknown data type, we should know it to enhance the method accordingly...I am affraid a log with info level will be missed easily...